pkg/k8s/k8s.go

```diff
@@ -35,12 +34,7 @@ const (
 	defaultHTTPSHostNetworkPort = int32(443)
 	defaultHTTPSPortName        = "https"
 
-	// Path and port to the healthcheck service
-	healthcheckPort        = 59999
-	healthcheckPath        = "/healthcheck"
-	healthcheckSidecarName = "nginx-healthchecker"
-
-	defaultSidecarContainerImage = "tsuru/nginx-operator-sidecar:latest"
+	curlProbeCommand = "curl -m20 -kfsS -o /dev/null %s"
```

Comment:
Should not this timeout value be less than `readinessProbe.timeoutSeconds` value?

Reply:
I'm not really sure we even need to bother with `-m` here. Since I'm pretty sure kubernetes will kill the curl process after `readinessProbe.timeoutSeconds`. I'll check this.

Reply:
Yeah, based on some tests it looks like I was wrong, the old exec probe is not killed.